src/utils/Cargo.toml

```diff
@@ -18,10 +18,10 @@ versionize = "0.1.10"
 versionize_derive = "0.1.6"
 vmm-sys-util = "0.11.2"
 vm-memory = { version = "0.13.0", features = ["backend-mmap", "backend-bitmap"] }
-log-instrument = { path = "../log-instrument", optional = true }
+log-instrument = { path = "../log-instrument" }
 
 [dev-dependencies]
 serde_json = "1.0.99"
 
 [features]
-tracing = ["log-instrument"]
+tracing = []
```

Comment:
commit message `fix: Fix optional dependency issue`: would be helpful to reference log-instrument somehow in the summary.

Reply:
I've updated the commit message to reference it:
```text
When running `cargo build --features tracing` with `src/utils`
instrumented errors are produced due to the `log-instrument` optional
dependency not being enabled by the `utils/tracing` feature. This commit
makes the dependency non-optional to fix this, when unneeded the
dependency will still be ommited at compile time however this may very
slightly increase compile time.
```